lib/config/config-file.js

```diff
@@ -156,7 +157,7 @@ function loadLegacyConfigFile(filePath) {
 function loadJSConfigFile(filePath) {
     debug(`Loading JS config file: ${filePath}`);
     try {
-        return importFresh(filePath);
+        return lodash.cloneDeep(importFresh(filePath));
```

Comment:
This change isn't needed. The only place that calls this function already clones it, so this would just create another copy of it.

Reply:
Yes the Ajv will still mutate the object though we use `lodash.cloneDeep` here, so I changed to use `lodash.cloneDeep` in `CLIEngine`.

Reply:
Ah, I see. Thanks for the explanation.